app/models/dashboard_presenter.rb

```diff
@@ -1,17 +0,0 @@
-class DashboardPresenter
-  def uncontacted_companies
-    Company.where(status: "uncontacted").order(updated_at: :desc)
-  end
-
-  def contacted_companies
-    Company.where(status: "contacted").order(updated_at: :desc)
-  end
-
-  def confirmed_companies
-    Company.where(status: "confirmed").order(updated_at: :desc)
-  end
-
-  def dead_companies
-    Company.where(status: "dead").order(updated_at: :desc)
-  end
```

Comment:
I guess...there will be merge conflict because yesterday, Laura and I did a lot of things with Dashboard presenter